### DISCUSSION TITLE
expense project using ansible mysql_root_password error

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1868] [Q&A]
![Screenshot (42)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/b89b90b3-559f-4d14-b0fa-cb7197d93aa7)

**Comment by chaithanya1812** [comment]
Hello @Bhashitha19, 
Your doubt is not cleared ?? In lab session, I asked you to join after sometime ,But you never joined?
 ## This was your code, I run using ansible-pull, for me it's not show's any error!!

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/c9b8b7fe-6662-4d2f-b1f0-43d47918a295)
